packages/playwright-core/src/server/chromium/crPage.ts

```diff
@@ -587,7 +587,7 @@ class FrameSession {
   }
 
   async _navigate(frame: frames.Frame, url: string, referrer: string | undefined): Promise<frames.GotoResult> {
-    const response = await this._client.send('Page.navigate', { url, referrer, frameId: frame._id });
+    const response = await this._client.send('Page.navigate', { url, referrer, frameId: frame._id, referrerPolicy: 'unsafeUrl' });
```

Comment:
Why was it not required before and now we have to add it?

Reply:
By default it has [network::mojom::ReferrerPolicy::kDefault](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/devtools/protocol/page_handler.cc;l=462;drc=2d0a33ccdfa8ac1a9bdf1d86d3b9138f99484bdf), and if the referer was cross-origin, it did not pass it over. Only the same origin was working. After the change it works cross-origin and align it with the other browsers.